mllib/src/main/scala/org/apache/spark/ml/feature/StopWordsRemover.scala

```diff
@@ -73,22 +75,38 @@ class StopWordsRemover(override val uid: String)
   /** @group getParam */
   def getCaseSensitive: Boolean = $(caseSensitive)
 
-  setDefault(stopWords -> StopWordsRemover.loadDefaultStopWords("english"), caseSensitive -> false)
+  /**
+   * Locale for doing a case sensitive comparison
+   * Default: English locale ("en")
+   * @see [[http://www.localeplanet.com/java/]]
```

Comment:
Please link to the official Java doc: https://docs.oracle.com/javase/8/docs/api/java/util/Locale.html or the `Locale` class.